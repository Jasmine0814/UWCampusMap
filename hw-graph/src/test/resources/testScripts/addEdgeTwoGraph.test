# test that create 2 graphs and add two pair of nodes to each graph
# and add two edge between each pair of nodes

# Create Graphs
CreateGraph graph1
CreateGraph graph2

# Add Nodes to the graphs
AddNode graph1 n1
AddNode graph1 n2
AddNode graph2 n3
AddNode graph2 n4

# Add Edges to the graphs
AddEdge graph1 n1 n2 e1
AddEdge graph2 n3 n4 e2

# Print the nodes in the graph and the edges
ListNodes graph1
ListChildren graph1 n1
ListNodes graph2
ListChildren graph2 n3
