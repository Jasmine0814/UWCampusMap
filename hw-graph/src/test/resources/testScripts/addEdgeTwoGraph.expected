# test that create 2 graphs and add two pair of nodes to each graph
# and add two edge between each pair of nodes

# Create Graphs
created graph graph1
created graph graph2

# Add Nodes to the graphs
added node n1 to graph1
added node n2 to graph1
added node n3 to graph2
added node n4 to graph2

# Add Edges to the graphs
added edge e1 from n1 to n2 in graph1
added edge e2 from n3 to n4 in graph2

# Print the nodes in the graph and the edges
graph1 contains: n1 n2
the children of n1 in graph1 are: n2(e1)
graph2 contains: n3 n4
the children of n3 in graph2 are: n4(e2)